### DISCUSSION TITLE
error in aws deploy

### DISCUSSION THREAD

**Discussion by aragornz325** [serverpod/serverpod#3069] [Q&A] [answered]
![image](https://github.com/serverpod/serverpod/assets/70964354/d80f2cea-aa7e-496d-bc94-bd59803b05b7)

Hello!... I configured everything as it is in the tutorial, but I get this error message and I don't know how to solve it. I tried to modify the script so that it loads version 3.0.0, but when deploying it always tries to deploy version 2.18.1

**Comment by vlidholt** [comment, marked as answer]
It looks like you need a later Dart version to correctly build your server (especially the `prlab_server package`). You can change the Dart version in the Github action, `.github/workflows/deployment-aws.yaml`.
